py/dict.go

```diff
@@ -36,6 +36,29 @@ func init() {
 		}
 		return NewIterator(o), nil
 	}, 0, "items() -> list of D's (key, value) pairs, as 2-tuples")
+
+	StringDictType.Dict["get"] = MustNewMethod("get", func(self Object, args Tuple) (Object, error) {
+		var length = len(args)
+		if length == 0 {
+			return nil, ExceptionNewf(TypeError, "%s expected at least 1 arguments, got %d", "items()", length)
+		} else if length > 2 {
+			return nil, ExceptionNewf(TypeError, "%s get expected at most 2 arguments, got %d", "items()", length)
+		}
+		sMap := self.(StringDict)
+		if str, ok := args[0].(String); ok {
+			res, ok := sMap[string(str)]
+			if ok {
```

Comment:
ditto:
```go
switch res, ok := sMap[string(str)]; ok {
case ok:
    return res, nil
case length == 1:
    return None, nil
default:
    return args[1], nil
}
```
```

Reply:
@sbinet 
``` go
switch res, ok := sMap[string(str)]; ok {
case ok:
    return res, nil
case length == 1:
    return None, nil
default:
    return args[1], nil
}
```
I applied as you advised, but it caused a "panic".
So I made some modifications.